pkg/solidity-utils/contracts/openzeppelin/SafeERC20.sol

```diff
@@ -19,6 +19,20 @@ import "@balancer-labs/v2-interfaces/contracts/solidity-utils/openzeppelin/IERC2
  * which allows you to call the safe operations as `token.safeTransfer(...)`, etc.
  */
 library SafeERC20 {
+
+    function safeApproval(
```

Comment:
I'd rename this to `safeApprove`

Reply:
I got `safeApproval` from somewhere I can't remember now, but it did sound weird :)
Done.